src/components/bodyBackdrop/BodyBackdrop.jsx

```diff
@@ -0,0 +1,10 @@
+import ReactDOM from "react-dom";
+
+import "./css/bodyBackdropStyle.css";
+
+export default function BodyBackdrop(props) {
+  return ReactDOM.createPortal(
```

Comment:
I received a feedback from an MA earlier that we don't really require portal for the backdrop. We can create at the root level.